scripts/genAiKey.js

```diff
@@ -0,0 +1,15 @@
+const fs = require("fs");
+
+if (process.env.TRAVIS_TAG) {
+    const ISPROD = /^v?[0-9]+\.[0-9]+\.[0-9]+$/.test(process.env.TRAVIS_TAG || "");
+    const packageJson = JSON.parse(fs.readFileSync("package.json"));
+    if (ISPROD) {
+        packageJson.aiKey = process.env["PROD_AIKEY"];

```

Comment:
when do we set the env?

Reply:
It is set in Travis: https://travis-ci.org/Microsoft/vscode-azure-iot-toolkit/settings